docs/build/get-started/_category_.json

```
@@ -0,0 +1,6 @@
+{
+  "label": "Get Started",
```

```suggestion
  "label": "Get started",
```

We follow the Google developer documentation style guide for heading format, which is to use sentence case. 😀

Sentence case is a capitalization style where only the first letter of the first word, proper nouns, and other specific conventions are capitalized, while the rest of the text remains in lowercase.

Here's more info, if helpful! https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings